docs/open/QUICKSTART.md

```diff
@@ -209,15 +231,15 @@ You should see now 4 total nodes connected like below via the DC/OS UI.
 
 
 # Upgrading Your Cluster
-Next, let’s upgrade our cluster.
+Terraform also makes it easy to upgrade our cluster to a newer version of DC/OS.
+If you are interested in learning more about the upgrade procedure that Terraform performs, please see the official [DC/OS Upgrade documentation](https://docs.mesosphere.com/1.11/installing/production/upgrading/). 
 
-You can use DC/OS Terraform to not only install, but to maintain and upgrade your cluster to newer versions of DC/OS quickly and effortlessly.
+1) In order to perform an upgrade, we need to go back to our `main.tf` and specify an additional parameter (`dcos_install_mode`). By default this parameter is set to `install`, which is why we were able to leave it unset when creating the initial DC/OS cluster and scaling it. 
 
-Let’s go back to our `main.tf` and specify an additional parameter.  You can check the many parameters available for you to add in `main.tf` to get the most power out of DC/OS Terraform [here](http://registry.terraform.io/modules/dcos-terraform/dcos/aws/). In the meantime, we’ll just focus on one - `dcos_install_mode`.
+Since we’re now upgrading, however, we need to set this parameter to `upgrade`.
 
-Since we’re upgrading, we set this parameter to `upgrade`. 
+**NOTE:** We do not actually upgrade DC/OS to a newer version during this procedure. We simply upgrade it in place to the same version in order to demonstrate how it can be done.
```

Comment:
Is this actually true? If not, I don't see how / where we specify an upgrade to, say, 1.11.5. Maybe we should perform an *actual* upgrade here -- not a fake one.

Reply:
if we go with fixed version in this example we will be able to do such an upgrade